src/options.h

```diff
@@ -113,9 +113,9 @@
 #  define MEMDEBUG
 #  define KEEP_BODIES
 #  endif
-// Memory sanitizer also needs thread-local storage
+// Memory sanitizer also needs small memory model
```

Comment:
Both of which are required for TLS. I'd rather have a flag that occasionally segfaults but otherwise tells me where memory bugs are than one that doesn't work at all.